core/lexicon/en/workspace.inc.php

```diff
@@ -23,14 +23,14 @@
 $_lang['curl_not_installed'] = 'cURL must be installed and configured as an extension for PHP for you to use Package Management. You can find more information about cURL on the <a href="http://php.net/curl" target="_blank">PHP cURL website</a>.';
 $_lang['description'] = 'Description';
 $_lang['details'] = 'Details';
-$_lang['dir_err_assets'] = 'Your assets/ directory was not able to be created at: [[+path]] <br />You will need to create this directory and make it writable if you want to use Package Management or 3rd-Party Components.';
-$_lang['dir_err_assets_comp'] = 'Your assets/components/ directory was not able to be created at: [[+path]] <br />You will need to create this directory and make it writable if you want to use Package Management or 3rd-Party Components.';
-$_lang['dir_err_core_comp'] = 'Your core/components/ directory was not able to be created at: [[+path]] <br />You will need to create this directory and make it writable if you want to use Package Management or 3rd-Party Components.';
+$_lang['dir_err_assets'] = 'Your assets/ directory was not able to be created at: [[+path]] <br />You will need to create this directory and make it writable if you want to use Package Management or 3rd-Party Extras.';
+$_lang['dir_err_assets_comp'] = 'Your assets/components/ directory was not able to be created at: [[+path]] <br />You will need to create this directory and make it writable if you want to use Package Management or 3rd-Party Extras.';
+$_lang['dir_err_core_comp'] = 'Your core/components/ directory was not able to be created at: [[+path]] <br />You will need to create this directory and make it writable if you want to use Package Management or 3rd-Party Extras.';
 $_lang['download'] = 'Download';
-$_lang['download_success'] = 'The Extra was downloaded successfully!';
+$_lang['download_success'] = 'The Package was downloaded successfully!';
 $_lang['downloaded'] = 'Downloaded';
 $_lang['downloading'] = 'Downloading...';
-$_lang['download_extras'] = 'Download Extras';
+$_lang['download_extras'] = 'Download Packages';
```

Comment:
Same as above.

Reply:
It looks weird and unfinished. When 1 entity in neighboring buttons is called differently.

![package](https://user-images.githubusercontent.com/12523676/150843883-ff33fe11-2715-4708-9b14-ce3f05353db6.png)